src/components/ErrorList/ErrorList.tsx

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+
+interface Props {
+  errors: string;
+  onClear: () => void;
+}
+
+export const ErrorList: React.FC<Props> = ({ errors, onClear }) => {
+  useEffect(() => {
+    if (errors.length) {
```

Comment:
you can make it shortest
```suggestion
    if (errors) {
```